src/Data/Hashable/Class.hs

```diff
@@ -906,6 +906,8 @@ instance Hashable a => Hashable (Max a) where
     hashWithSalt p (Max a) = hashWithSalt p a
 
 -- | __Note__: Prior to @hashable-1.3.0.0@ the hash computation included the second argument of 'Arg' which wasn't consistent with its 'Eq' instance.
+-- Additionally, as of @hashable-1.3.0.0@, and going forward as part of the API guarantee, the hash of @'Arg' a _@ is the same value as the hash of @a@.
+-- This means that it's safe to add and remove 'Arg' from elements stored in hash-indexed data structures without changing any hashes.
```

Comment:
Ok, so this is actually not true.

`hashWithSalt salt (Arg x _) = hashWithSalt x` by definitiuon, but `hash (Arg x _) /= `hash x`:

```haskell
> hash (Nothing :: Maybe Int)
0
> hash (Arg (Nothing :: Maybe Int) 'x')
-5808590958014384161
```

That is an unfortunate "oversight".